addons/website/static/tests/tours/link_tools.js

```diff
@@ -44,6 +44,15 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
         trigger: '#o_link_dialog_url_input',
         run: 'text odoo.be'
     },
+    {
+        content: "Url value should contain odoo.be",
+        trigger: '#o_link_dialog_url_input',
+        run: () => {
+            if ($('#o_link_dialog_url_input').val() !== 'odoo.be') {
+                throw new Error('URL value should contain odoo.be');
+            }
+        },
```

Comment:
In general, we do `console.error` inside tests instead of `throw new Error`. However in this case, this is probably the same and should thus be written that way:
```suggestion
        trigger: '#o_link_dialog_url_input:propValue(odoo.be)',
        run: () => null,
```

However, I don't think this added step makes sense: the previous step says "add odoo.be in the input", then without unfocus or anything, you say "check that odoo.be is in the input"... AFAIK it cannot ever be wrong. More than this, I tested here: adding this step without your fix does not make the tour break (tested in both 15.0 and 16.0) => so this is not testing anything, it is something that always passes.